app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,13 @@
+module SessionsHelper
```

Comment:
this is probably where a "log out" method would go that would be triggered by the controller